projects/plugins/jetpack/_inc/client/at-a-glance/backups.jsx

```diff
@@ -102,6 +102,71 @@ class DashBackups extends Component {
 		} );
 	};
 
+	getJetpackBackupBanner() {
+		const { partnerCoupon, upgradeUrl, siteRawUrl, trackUpgradeButtonView } = this.props;
+
+		if ( this.props.hasConnectedOwner ) {
+			if ( partnerCoupon ) {
+				const checkoutUrl = getRedirectUrl( 'jetpack-plugin-partner-coupon-checkout', {
+					path: partnerCoupon.product.slug,
+					site: siteRawUrl,
+					query: `coupon=${ partnerCoupon.coupon_code }`,
+				} );
```

Comment:
The other upgrade cards will redirect to an expanded product description before going through checkout.
Do we know if we should do the same here and redirect users to the bigger "Redeem coupon" component with some more information about backups, before going to checkout?

The product description step reduced refund rate quite a bit, but seeing as this is free backups... hmm... I'm not sure if it's an issue or not that we don't highlight product features before checkout 😅

Reply:
The bigger CTA we have isn't much more informative IMO so we're in a position where we either:
- Show very limited information and a Redeem CTA that leads to checkout
- or, a Redeem CTA that leads to a slightly more informative Redeem CTA again which then leads to checkout

I don't see Design stating which way this should go and this affects both of our PRs so I've reached out in pdpAdu-1U-p2#comment-142.

Reply:
For posterity: Design recommended we redirect to checkout.